zig/download.bat

```diff
@@ -60,33 +60,11 @@ if exist %ZIG_TARBALL% (
 echo Downloading %ZIG_URL%...
 curl --silent --progress-bar --output %ZIG_TARBALL% %ZIG_URL%
 if not exist  %ZIG_TARBALL% (
-  echo Failed to download Zig zip file.
+  echo Failed to download zip file.
   exit 1
 )
 
-:: Replace any existing Zig installation so that we can install or upgrade:
```

Comment:
Out of scope for this PR, but it would be great to get rid of this whole bat script and replace it with a powershell one.

Reply:
Indeed!